### DISCUSSION TITLE
Is Inventree usable as a "library" of inventory

### DISCUSSION THREAD

**Discussion by SmrtSmrtnik** [inventree/InvenTree#7902] [General]
Hello,
I am reading through the documentation and played around with the demo a bit, but I'm not clear.
Is it possible to customize/use Inventree as a tracker of internal inventory, for example a fleet of laptops that can be checked in and out of a lab (to specific people)?

**Comment by matmair** [comment]
You could but I would recommend looking into snipe-it for this use case.https://docs.inventree.org/en/stable/concepts/terminology/

**Reply by SmrtSmrtnik** [reply]
perfect, thank you so much!
